nova_vm/src/heap/heap_bits.rs

```diff
@@ -54,33 +82,105 @@ pub struct WorkQueues {
 
 impl HeapBits {
     pub fn new(heap: &Heap) -> Self {
+        let mut modules = Vec::with_capacity(heap.modules.len());
```

Comment:
If I understand correctly, we should be able to do this:

```rust
let modules = vec![false; heap.modules.len()];
```

instead of this:

```rust
let mut modules = Vec::with_capacity(heap.modules.len());
modules.resize(modules.capacity(), false);
```

for all of these.

Reply:
I knew there had to be a better way, I just couldn't remember anything :D